smartcosmos-framework/src/main/java/net/smartcosmos/aspects/SmartCosmosServiceAspect.java

```diff
@@ -14,8 +16,8 @@
 /**
  * Aspect to be applied to extension services implemented in the SMART COSMOS Objects architecture.
  * <p>
- * Using this aspect will capture all exceptions and convert them into a {@see SmartCosmosException}.  If your methods do not declare
- * they throw SmartCosmosException then you will receive an {@see UndeclaredThrowableException}
+ * Using this aspect will capture all exceptions and convert them into a {@link SmartCosmosException}.  If your methods do not declare
+ * they throw SmartCosmosException then you will receive an {@link UndeclaredThrowableException}
```

Comment:
I've had bad experiences where documentation references code in `{@link } ` then when a class is changed or deprecated the javadocs cause a build failure